### ISSUE TITLE
Meeting Notes 2/6

### ISSUE THREAD

**Issue by panalluri** [ocean-transport/air-sea-ML#18]
Diagnostics project:
- The dropoff for each filter should go almost vertically down after the filter cutoff <-- this is not happening
- Try the trial examples w GCM filters in the notebook link Dhruv sent --> apply what you learned to fix my graphs
- Use whiteboard pic to derive power spectra units w/ xrft
- Image in GCM filters documentation plots the filter kernel, not the filter cutoff like I thought
- Play with the GCM theory notebook in documentation --> plot the filter cutoff on a log scale
- air-sea flux anomalies --> don't use imshow
    - plot with lat-lon axes
    - the current plots are a mirror image of the earth
    - plot for some specific regions, not just the whole earth (e.g. gulf stream region, eastern pacific region off west coast of south america, 1000 x 1000 km)
    - narrow in on the colobar scale for freshwater flux (e.g. use plt.plot, robust=True)
    - make movies of the flux anomalies

Super-res project:

custom alpha loss:
- dig into every single line in CNN code + custom loss function w/ alpha and training loop and understand why it works
- minimum working example: try custom loss function alpha with a dummy MSE loss implementation + very simple data to try to debug (e.g. linear regression) --> this allows us to isolate the loss issue, and not get mixed into the CNN issues

fft loss:
- document what happens when only fft MSE is in loss func (no real space MSE)
- determine whether we need MSE for magnitude and phase or not

Overleaf:
- add board pics from last 3 meetings
- document results of each experiment

Paper:
- describe tianyi li et al paper to dhruv --> make a short ppt (45 mins ppt)
